styles/alphav3-1.json

```diff
@@ -11,122 +11,122 @@
       "palette": [
         {
           "slug": "primary-brand",
-          "color": "var(--bcds-surface-color-background-dark-blue)",
+          "color": "#053662",
```

Comment:
So we can't use the BCDS tokens here?

Reply:
well we could, but it would have to be the v2, as you can't run v2 and v3 parallel, and after my last issue of the tokens missing, it is better to just add the hex colors, so they aren't dependant on any tokens